packer/images.json.pkr.hcl

```diff
@@ -114,6 +114,7 @@ source "qemu" "cos-arm64" {
   disk_size              = "${var.disk_size}"
   format                 = "qcow2"
   headless               = true
+  net_bridge             = "virbr0"
```

Comment:
now... it starts to need this?! seriously?! :rofl:

Reply:
good catch btw!

Reply:
this could be because package by default uses user networking which on qemu is...complicated. Lots of variance in configs in the different distros and based on the configured system, so for all we know a simple hardware update could cause this or a newer qemu version, or a different network config for the system, etc...


Its kind of a pita :/